sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -18,7 +18,6 @@
 package org.apache.spark.sql
 
 import java.io.File
-import java.math.MathContext
```

Comment:
BTW, please do not remove the unnecessary import next time. Thanks!

When I backported it to 2.2, I hit a build break issue because it is still being used by the other test cases in 2.2

Reply:
@gatorsmile Oh.. OK.. Sorry about it. I will not remove next time.